app/controllers/checkout_controller.rb

```diff
@@ -1,13 +1,10 @@
 class CheckoutController < ApplicationController
   def index
-    if Order.last == nil
-      redirect_to root_path
-      flash[:notice] = "You have no items in your order"
-    elsif Order.last.shopping_cart_items.count == 0
+    if Order.where(bought: false).count == 0 || Order.find_by(bought: false).shopping_cart_items.count == 0
       redirect_to root_path
       flash[:notice] = "You have no items in your order"
     else
-      @order_items = Order.last.shopping_cart_items.all
+      @order_items = Order.find_by(bought: false).shopping_cart_items
```

Comment:
Right. When devise is set up. It is going say current_user.order.find_by(bought: false). Or is that complicating things? Maybe we can have a Pairprogramming session when fixing this? I know how we need to handle this but we can fix it together maybe